samples/tacos/src/main/kotlin/com/slack/circuit/tacos/OrderTacosCircuit.kt

```diff
@@ -0,0 +1,360 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.slack.circuit.runtime.CircuitContext
+import com.slack.circuit.runtime.CircuitUiEvent
+import com.slack.circuit.runtime.CircuitUiState
+import com.slack.circuit.runtime.Navigator
+import com.slack.circuit.runtime.Screen
+import com.slack.circuit.runtime.presenter.Presenter
+import com.slack.circuit.runtime.ui.Ui
+import com.slack.circuit.runtime.ui.ui
+import com.slack.circuit.tacos.model.Ingredient
+import com.slack.circuit.tacos.step.ConfirmationOrderStep
+import com.slack.circuit.tacos.step.ConfirmationProducer
+import com.slack.circuit.tacos.step.ConfirmationUi
+import com.slack.circuit.tacos.step.FillingsOrderStep
+import com.slack.circuit.tacos.step.FillingsProducer
+import com.slack.circuit.tacos.step.FillingsUi
+import com.slack.circuit.tacos.step.OrderStep
+import com.slack.circuit.tacos.step.SummaryOrderStep
+import com.slack.circuit.tacos.step.SummaryProducer
+import com.slack.circuit.tacos.step.SummaryUi
+import com.slack.circuit.tacos.step.ToppingsOrderStep
+import com.slack.circuit.tacos.step.ToppingsProducer
+import com.slack.circuit.tacos.step.ToppingsUi
+import com.slack.circuit.tacos.step.summaryProducer
+import java.math.BigDecimal
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.collections.immutable.persistentSetOf
+import kotlinx.collections.immutable.toImmutableSet
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+object OrderTacosScreen : Screen {
+  data class State(
+    @StringRes val headerResId: Int,
+    val orderCost: BigDecimal,
+    val stepState: OrderStep.State,
+    val isPreviousVisible: Boolean,
+    val isNextEnabled: Boolean,
+    val isNextVisible: Boolean,
+    val isFooterVisible: Boolean,
+    val eventSink: (Event) -> Unit
+  ) : CircuitUiState
+
+  sealed interface Event : CircuitUiEvent {
+    val value: Int
+      get() = 0
+
+    object Previous : Event {
+      override val value: Int = -1
+    }
+    object Next : Event {
+      override val value: Int = 1
+    }
+    object ProcessOrder : Event {
+      override val value: Int = 1
+    }
+  }
+}
+
+@Parcelize
+@Immutable
+data class OrderDetails(
+  val filling: Ingredient = Ingredient(""),
+  val toppings: Set<Ingredient> = persistentSetOf(),
+  val baseCost: BigDecimal = BigDecimal("9.99"), // Default taco price
+  val ingredientsCost: BigDecimal = BigDecimal.ZERO
+) : Parcelable
+
+private val orderSteps =
```

Comment:
nit: `ORDER_STEPS`

Reply:
I initially went with that, but it violates our lint rules